examples/website/src/shared/blocks/embedBlock.tsx

```diff
@@ -100,7 +100,7 @@ function embedForData(data: EmbedData) {
             allowFullScreen
             title={data.title}
             src={data.url}
-            width={data.width}
+            width="100%"
```

Comment:
In my opinion it should overtake data.width per default and if data.width is not set a.k. undefined use 100%
Like so:
`width={data.wdith || "100%"}`

Reply:
Actually you are right it doesn't make sense to override the data. There was an underlying issue here which was causing issues with the width -it was saved as number format and therefore lost the '%' sign when rendering. Iframe with 100% width got saved as 100px width. I restored the original `data.width` here and it should be working.